src/components/DocumentationTopic/Summary/ListItem.vue

```diff
@@ -35,11 +35,11 @@ export default {
 
 <style scoped lang="scss">
 .summary-list-item {
-  margin-bottom: 0.25rem;
+  margin-right: 8.5px;
```

Comment:
8.5px is very specific and we barely don't use any decimals in the rest of the code. Can we change it to 8px or 9px instead?

Reply:
Changed to 9px.